test/helpers.js

```diff
@@ -0,0 +1,19 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: strong-tools
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var rimraf = require('rimraf');
+var fs = require('fs');
+
+exports.resetSandboxSync = resetSandboxSync;
+
+function resetSandboxSync(t, sandbox, sandboxPkg, pkgjson) {
+  rimraf.sync(sandbox);
+  fs.mkdirSync(sandbox);
```

Comment:
Hm. It isn't necessary in the code that this was extracted from, but you make a good point in the context of a more generic utility function.